rs/sns/governance/src/proposal.rs

```diff
@@ -1412,6 +1412,10 @@ pub fn validate_and_render_add_generic_nervous_system_function(
         return Err("Reached maximum number of allowed GenericNervousSystemFunctions".to_string());
     }
 
+    if validated_function.topic.is_none() {
```

Comment:
Add a comment explaining why this validation is not in ValidGenericNervousSystemFunction::try_from